How much time is left per drug feature

stephaniege opened the pull request.

Commit:
Controller code only for feature for seeing estimates for how much time left per drug.

seshness commented:
Hold up; this should be in the model. A `Drug#time_left` method would suffice for capturing this logic.

Making this change now..

seshness committed:
merge master into this branch

Summary:
Abstract out time_left into Drug model.

seshness committed:
make like master branch - no edit intended to pharmacy_controller

seshness committed:
scratch

seshness committed:
[pharmacy] Display time remaining for drugs

Summary:
Based on user_rate if available, or estimated_rate as calculated from previous
week.

seshness committed:
correctly display estimated rate of usage
